test/benchmark/iperf3/iperf3_bench_runner.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
I am not sure. How will nginx be evaluated? If it has similar process, we can rename this script to `host_guest_bench_runner.sh` and move it to `common/`.

Reply:
I think the only difference between iperf3 and nginx is the sleep time. However, using a 2-minute sleep time for Asterinas may also work well in the Nginx benchmark.

Well, I think it's okay to keep the `iperf3_bench_runner.sh`. I will rename the scripts when necessary since I will add the Nginx and the Redis benchmark CI.